src/archipelago/apdoom.cpp

```diff
@@ -407,7 +408,7 @@ int apdoom_init(ap_settings_t* settings)
 	max_map_count = 0; // That's really the map count
 	for (const auto& episode_level_info : level_info_table)
 	{
-		max_map_count = max(max_map_count, (int)episode_level_info.size());
+		max_map_count = std::max(max_map_count, (int)episode_level_info.size());
```

Comment:
You were using the C max symbol, which does not exist on linux. Changed to the c++ one which is part of the standard library